release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -268,6 +268,41 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
 [id="ocp-4-8-monitoring"]
 === Monitoring
 
+[id="ocp-4-8-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.8 includes the following alerting rule changes:
+
+.Alerting rule changes
+[%collapsible]
+====
+* The `ThanosSidecarBucketOperationsFailed` alert is added. The warning alert provides notification if the Thanos sidecar bucket operations are failing.
+* The `ThanosSidecarPrometheusDown` alert severity is updated from _critical_ to _warning_.
+* The `ThanosSidecarUnhealthy` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryHttpRequestQueryErrorRateHigh` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryHttpRequestQueryRangeErrorRateHigh` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryInstantLatencyHigh` critical alert is removed. This alert fired if Thanos Querier had a high latency for instant queries.
+* The `ThanosQueryRangeLatencyHigh` critical alert is removed. This alert fired if Thanos Querier had a high latency for range queries.
+* For all Thanos Querier alerts, the `for` duration is increased to 1 hour.
+* For all Thanos sidecar alerts, the `for` duration is increased to 1 hour.
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for metrics, recording rules, or alerting rules.
+=====
+====
+
+[id="ocp-4-8-monitoring-kube-state-metrics-2.0.0-upgrade"]
+==== kube-state-metrics upgraded to version 2.0.0-rc.1
+
+`kube-state-metrics` is upgraded to version 2.0.0-rc.1. For more information on all changes in this version, including updates to metrics and flags, see the link:https://github.com/kubernetes/kube-state-metrics/blob/master/CHANGELOG.md[kube-state-metrics CHANGELOG].
+
+[id="ocp-4-8-monitoring-recording-rules-added-builds-by-strategy"]
+==== Recording rules added for builds by strategy
+
+Recording rules are added that aggregate the number of builds running on a cluster by strategy.
+// CHECK if this note should also include the following specific information: These rules produce a time series with strategy labels of `docker`, `source`, `jenkinspipeline`, and `custom`.
```

Comment:
How would this get rendered?

Reply:
@sthaha Only the first sentence, "Recording rules are added ...", is rendered right now. I left a comment/note (`//`) for myself to check if more information needs to be included in the release note, i.e., "These rules produce a time series ..." 👍